Automattic/jetpack
# General: update readme.txt

#### Changes proposed in this Pull Request:

* Updated plugin description

#### Jetpack product discussion

* pbNhbs-eE-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Please check for any spelling, grammar, or technical errors. 
* Double check the formatting is correct and presents correctly in context in the plugin and on WP.org.

#### Proposed changelog entry for your changes:

* N/A
